x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/sync_rules/advanced_sync_rules.tsx

```diff
@@ -40,7 +40,7 @@ export const AdvancedSyncRules: React.FC = () => {
       }
       fullWidth
     >
-      <CodeEditor
+      <CodeEditor // TODO theme is provided in context by x-pack/plugins/enterprise_search/public/applications/index.tsx
```

Comment:
What is the actual TODO here? It's unclear to me what action needs to be taken.

Reply:
Heya @sphilipse ! The `CodeEditor` marked with the TODO should exist somewhere within a plugin owned by your team (Hopefully I got it right!). Being unfamiliar with where all the `CodeEditor` fields exist in the app, I'm asking the owners to help me verify that their `CodeEditor`s are displaying as expected.

Please enable Dark Mode from Edit User Profiles by clicking on the Profile Icon on the top right, and updating your profile.

Navigate to the CodeEditor mentioned above in the app. If they render and display in Dark Mode - add a value to the `Verified Working` table in the PR description  - and you're done!

If it is not rendering, please help me figure out where the theme service should be provided in the context.